<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CitiesWidget</class>
 <widget class="QWidget" name="CitiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cities</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="countryComboBox"/>
   </item>
   <item>
    <widget class="QComboBox" name="stateComboBox"/>
   </item>
   <item>
    <widget class="QPushButton" name="newCityBtn">
     <property name="text">
      <string>New</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>countryComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CitiesWidget</receiver>
   <slot>setState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newCityBtn</sender>
   <signal>clicked()</signal>
   <receiver>CitiesWidget</receiver>
   <slot>newCity()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stateComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>CitiesWidget</receiver>
   <slot>updateTable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setState()</slot>
  <slot>newCity()</slot>
  <slot>updateTable()</slot>
 </slots>
</ui>
